wai-aria/heading/invalid-nesting.tentative.html

```diff
@@ -0,0 +1,153 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>WAI-ARIA tests for invalidly nested elements with heading role</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+
+<body>
+
+  <h1>WAI-ARIA tests for invalidly nested heading elements</h1>
```

Comment:
@adampage It may also be useful to add tests with overridden roles (e.g., `<button role="heading">` or `<p role="heading">`).